tools/train.py

```diff
@@ -127,10 +129,21 @@ def main():
                     loss *= ctx.get_loss_scale_var()
                 lr = lr_builder()
                 optimizer = optim_builder(lr)
-                optimizer.minimize(loss)
+                clip = None
+                if optim_builder.clip_grad_by_norm is not None:
+                    clip = fluid.clip.GradientClipByGlobalNorm(
+                        clip_norm=optim_builder.clip_grad_by_norm)
+                optimizer.minimize(loss, grad_clip=clip)
```

Comment:
In develop, grad_clip is moved to init in optimizer

Reply:
will this change land in 2.0?